pam-u2f.c

```diff
@@ -279,6 +279,25 @@ int pam_sm_authenticate(pam_handle_t *pamh, int flags, int argc,
     should_free_auth_file = 1;
     buf = NULL;
   } else {
+    if (strncmp(cfg->auth_file, "/", 1) != 0) { /* Individual authorization mapping by user: auth_file is not absolute path, so prepend user home dir. */
```

Comment:
I would do cfg->auth_file[0] != '/' here, no need calling a libc function

Reply:
Fine with me. As strncmp() is in use when parsing options I figured it would do no harm to use it here, though.

If you want me to drop the strncmp() anyway, just confirm briefly and I'll rework the PR.